tests/plugins/inlinetoolbar/_helpers/default.js

```diff
@@ -0,0 +1,19 @@
+/* exported replaceAppendStyleSheet, convertRgbaToRgb */
+
+'use strict';
+
+function replaceAppendStyleSheet() {
```

Comment:
This looks good, however it doesn't provide a possibility to clean up after running tests (restore original function). You could do two things:

* This function might return a function which will restore original one when called:
```
return function() {
    CKEDITOR.dom.document.prototype.appendStyleSheet = oldFn;
}
```
* Use sinon stub which has `stub.restore()` method.

It is good practise to always clean up after test run (especially for unit tests, for manual you may skip it). So after changing it, please add clean up to unit test (you may use `tearDown` function).

Reply:
I decided to not use `sinon.stub`, because I wasn't able to find possibility to fire origin function within stub.
In documentation I found only possibility how to replace the method, with own function.
That's why I wrote some similar wrapper which could execute native method. I'll extend it a little bit to restore native method.

Reply:
It works fine but it's a little over complicated for me. You have to create new instance, it has two methods which you have to call, etc. plus you reinvented a wheel (well, a part of it:D).

> In documentation I found only possibility how to replace the method, with own function.

That's exactly where you can call the original function, the stub could look like this:

```
function stubAppendStyleSheet() {
	var originFn = CKEDITOR.dom.document.prototype.appendStyleSheet;
	return sinon.stub( CKEDITOR.dom.document.prototype, 'appendStyleSheet', function( cssFileUrl ) {
		// Simulate missing css in skin.
		if ( !cssFileUrl.match( /(inlinetoolbar|balloonpanel)\.css/ ) ) {
			originFn.call( CKEDITOR.document, cssFileUrl );
		}
	} );
}
```

and then you can use it like:
```
var stub = stubAppendStyleSheet();
CKEDITOR.once( 'instanceReady', stub.restore );
```

It is shorter and simpler IMHO. Plus for everybody who knows `sinon`, you just take a look at the usage and it should be clear that these two lines stubs `appendStyleSheet` method and then restores it. With custom code you always have to dive in to check what's going on.